pkg/resourcecreator/deployment/deployment_test.go

```diff
@@ -10,7 +10,8 @@ import (
 
 	"github.com/nais/naiserator/pkg/resourcecreator/resource"
 
-	nais "github.com/nais/liberator/pkg/apis/nais.io/v1alpha1"
+	nais_io_v1 "github.com/nais/liberator/pkg/apis/nais.io/v1"
+	nais_io_v1_alpha1 "github.com/nais/liberator/pkg/apis/nais.io/v1alpha1"
```

Comment:
Hva er grunnen til at vi bruker både v1 og v1alpha1 her?

Reply:
Application er fortsatt på `v1alpha1`, mens vi starter med `v1` for Naisjob. Og de deler en del felleskode, så ting er litt delt plassert.